ts/state/selectors/conversations.ts

```diff
@@ -421,20 +421,23 @@ export const _getConversationComparator = (testingi18n?: LocalizerType) => {
 export const getConversationComparator = createSelector(getIntl, _getConversationComparator);
 
 // export only because we use it in some of our tests
+// tslint:disable-next-line: cyclomatic-complexity
 export const _getLeftPaneLists = (
   lookup: ConversationLookupType,
   comparator: (left: ReduxConversationType, right: ReduxConversationType) => number,
   selectedConversation?: string
 ): {
   conversations: Array<ReduxConversationType>;
   contacts: Array<ReduxConversationType>;
+  conversationRequests: Array<ReduxConversationType>;
```

Comment:
I already said it on another files but I think you have to get this to another selector.
It will be easier to maintain that way